Plans

```diff
@@ -0,0 +1,9 @@
+right and left scroll - will be in 2.5
```

Comment:
Plans are not your call to make. If these are PR plans, mark your PR as a draft and add the plans to the PR description.